pkg/flags/parse.go

```diff
@@ -42,5 +63,13 @@ func Parse() Flags {
 	if len(*whitelist) > 0 {
 		tempFlags.WhiteList = strings.Split(*whitelist, ",")
 	}
+	if *askForPass {
```

Comment:
could you make this a simple flag rather than introducing another prompt? i.e. add an optional `-password` flag

the prompt doesn't add much value and makes it harder to reuse existing dbdot commands from bash history or share it with someone else.